docs/csharp/codedoc.md

```diff
@@ -2,26 +2,547 @@
 title: Documenting your code
 description: Documenting your code
 keywords: .NET, .NET Core
-author: dotnet-bot
+author: tsolarin
 manager: wpickett
-ms.date: 06/20/2016
+ms.date: 07/24/2016
 ms.topic: article
 ms.prod: .net-core
 ms.technology: .net-core-technologies
 ms.devlang: dotnet
 ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 ---
 
-# 🔧 Documenting your code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/494) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# XML Documentation
+
+XML documentation tags are specially formatted comments, added above the definition of any user defined type or member, used to document your code.
+Unlike regular C# comments documentation comments are not ignored by the compiler and you can generate the XML documentation file at compile time
+by adding `"xmlDoc": true` under `buildOptions` in your `project.json` when using .NET Core or use the `/doc` compiler option for the .NET framework.
+Go [here](https://msdn.microsoft.com/en-us/library/3260k4x7.aspx) to learn how to enable XML documentation generation in Visual Studio.
+
+Documentation tags are commented using three forward slashes (`///`) as opposed to two forward slashes (`//`) used for single line comments.
+There are tools tools like [DocFX](https://dotnet.github.io/docfx/) and [Sandcastle](https://github.com/EWSoftware/SHFB) that help you generate documentation websites from your documentation tags,
+also, distributing the compiler generated XML file along with your library allows Visual Studio and other IDEs to show quick information about types or members when performing intellisense. 
+
+<a name="summary"></a>
+## &lt;summary&gt;
+
+The &lt;summary&gt; tag contains the primary information of a class or method.
```

Comment:
Everywhere you use one of the tags, put them in the `<summary>` style, using the back-ticks. 

That way, you won't need to escape each instance.